acquia/acquia_cms
# ACMS-825: Added template for maintenance and update php page.

**Motivation**

Fixes #[ACMS-812](https://backlog.acquia.com/browse/ACMS-812)

Create maintenance page template and do some design so that all these page look similar in design:

- Install page
- Update page
- Maintenance page

**Proposed changes**

Provide brand new experience for acquia cms maintenance and update php page.

**Testing steps**

- Install site check newly created install page for acquia cms distribution.
- Install site, put site on maintenance mode and check newly created maintenance page for acquia cms distribution.
- Visit `/update.php` to check newly created update.php page experience.

**Merge requirements**
- [_Enhancement_] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
